README.md

```diff
@@ -79,3 +79,78 @@ fn main() {
     // ...
 }
 ```
+
+## In tests
+
+Tests can use the `env_logger` crate to see log messages generated during that test:
+
+```toml
+[dependencies]
+log = "0.3"
+
+[dev-dependencies]
+env_logger = "0.3"
+```
+
+```rust
+#[macro_use]
+extern crate log;
+
+fn add_one(num: i32) -> i32 {
+    info!("add_one called with {}", num);
+    num + 1
+}
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    extern crate env_logger;
+
+    #[test]
+    fn it_adds_one() {
+        env_logger::init().unwrap();
```

Comment:
Perhaps this could suggest something like:

```rust
let _ = env_logger::init();
```

That way multiple tests could call this and every test that has this will be guaranteed to have logging enabled?